HelpSource/Classes/NodeProxy.schelp

```diff
@@ -366,6 +366,14 @@ reset everything to nil, neutralizes rate/numChannels
 argument::fadeTime
 if a fadeTime is given, first fade out, then clear.
 
+method::schedAfterFade
+
+Calls a function after the fadeTime and server latency has passed. If the proxy specifies a code::quant:: value, this is adjusted to it.
```

Comment:
Generally looks good to me, except I'm afraid I have to be a bit fussy about:

> If the proxy specifies a code::quant:: value, this is adjusted to it.

Is the quant adjusted to the fade+latency, or the other way around? You really can't tell here which one is "this" and which one is "it."

Easy fix.

Reply:
thanks, I hope it is clearer now.

Reply:
> Calls a function after the fadeTime and server latency have passed. If the proxy specifies a code::quant:: value, this call happens relative to the specified beat.

To be honest, I had to go check the code to see what that means... for the second sentence, I might suggest:

If the proxy specifies a code::quant:: value, the function is evaluated `fadeTime + latency` seconds after the next timepoint on the `quant` grid; otherwise, the fade delay begins immediately.